CMakeLists.txt

```diff
@@ -100,6 +100,24 @@ if(NOT DEFINED WEBOTS_HOME)
   endif()
 endif()
 
+# If LD_LIBRARY_PATH is not already set, set it to the value of the LD_LIBRARY_PATH environment variable. If the environment
```

Comment:
This shouldn't be in this PR and needs to go before merging. Waiting on #88 to be merged since this won't work without it.